src/sagemaker_containers/encoders.py

```diff
@@ -38,17 +38,36 @@ def array_to_npy(array_like):  # type: (np.array or Iterable or int or float) ->
     return buffer.getvalue()
 
 
-def npy_to_numpy(npy_array):  # type: (object) -> np.array
+def npy_to_numpy(npy_array, dtype=np.float32):  # type: (object) -> np.array
     """Convert an NPY array into numpy.
 
     Args:
+        dtype (dtype, optional):  Data type of the resulting array.
```

Comment:
Should the docstrings for the args be reversed to match the function signature?

Same for other serializers.